docs/mkdocs.yml

```
@@ -167,6 +167,7 @@ nav:
       - 'Customize IMAP Folders': examples/use-cases/imap-folders.md
       - 'iOS Mail Push Support': examples/use-cases/ios-mail-push-support.md
       - 'Lua Authentication': examples/use-cases/auth-lua.md
+      - 'Bind outbound SMTP to a specific network': examples/user-cases/bind-smtp-network-interface.md
```

Typo fix. Sorry can't run the docs build locally atm so relying on CI 😬 

```suggestion
      - 'Bind outbound SMTP to a specific network': examples/use-cases/bind-smtp-network-interface.md
```